jbpm-test-coverage/src/test/java/org/jbpm/test/functional/CorrelationKeyTest.java

```diff
@@ -162,4 +166,37 @@ public void testMultiValuedKeyUniqueButInclusive() {
         ProcessInstance processInstance2 = ksession.startProcess(PROCESS, key2, null);
         assertProcessInstanceActive(processInstance2.getId());
     }
+    
+    @Test
+    public void testCreateProcessInstanceDefaultValue() {
+        CorrelationKey key = keyFactory.newCorrelationKey(SIMPLE_KEY);
+
+        Map<String, Object> parameters = new HashMap<String, Object>();            
+       
+        WorkflowProcessInstance processInstance = (WorkflowProcessInstance) ksession.startProcess(PROCESS, key, parameters);  
+        
+        assertEquals("defaultProc", processInstance.getVariable("procVar"));
+        assertEquals(Integer.valueOf(1), processInstance.getVariable("secondVar"));
+        
+    }    
+    
+    @Test
+    public void testCreateProcessInstanceComplexTypeDefaultValue() {
+       	String address = "<org.jbpm.test.domain.Address>\n"
+    			+ "          	   <street>abc</street>\n"
+    			+ "						   <number>29</number>\n"
+    			+ "						   <city>def</city>\n"
```

Comment:
Test failing because blanks shouldn't be present after tabs.
Anyway, I think this will also fail in windows due to a different separator. Better to use "xmlunit" library for this kind of comparison.

Reply:
Sure, updated the test cases accordingly.